diff -rupN src/mdns/mNameServer.h src_mod/mdns/mNameServer.h
--- src/mdns/mNameServer.h	2011-08-11 18:35:33.000000000 +0200
+++ src_mod/mdns/mNameServer.h	2011-08-11 23:32:54.212000047 +0200
@@ -63,10 +63,10 @@ namespace PracticaCaso
 			mNameServer(const mNameServer&);
 			mNameServer & operator = (const mNameServer &);
 			~mNameServer();
-			string mNameServer::translate(string dnsEntry);
-			void mNameServer::mdns_management(string cmd, string payload, string code); 
-			void mNameServer::mdns_manage_response(string cmd, string payload, string code); 
-			void mNameServer::mdns_manage_request(string cmd, string payload, string code); 
+			string translate(string dnsEntry);
+			void mdns_management(string cmd, string payload, string code); 
+			void mdns_manage_response(string cmd, string payload, string code); 
+			void mdns_manage_request(string cmd, string payload, string code); 
 			friend ostream & operator << (ostream &os, mNameServer &n);
 	};
 
